packages/webapp/src/containers/Navigation/index.jsx

```diff
@@ -21,12 +21,16 @@ import useIsFarmSelected from '../../hooks/useIsFarmSelected';
 import { CUSTOM_SIGN_UP } from '../CustomSignUp/constants';
 import useHistoryLocation from '../hooks/useHistoryLocation';
 import ReleaseBadgeHandler from '../ReleaseBadgeHandler';
+import { matchPath } from 'react-router-dom';
 
 const Navigation = ({ history, children, ...props }) => {
   const dispatch = useDispatch();
   const historyLocation = useHistoryLocation(history);
   const isCustomSignupPage = historyLocation.state?.component === CUSTOM_SIGN_UP;
   const isFarmSelected = useIsFarmSelected();
+  const isAcceptingInvite = ['/accept_invitation/sign_up'].some((path) =>
```

Comment:
`/accept_invitation/create_account` also needs to be added?

Reply:
Thanks!